Tests/SonarQube.TeamBuild.PreProcessor.Tests/BuildWrapperInstallerTests.cs

```diff
@@ -0,0 +1,116 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="BuildWrapperInstallerTests.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+using TestUtilities;
+
```

Comment:
Three scenarios to test:
1) no C++ plugin -> info message saying no integration
2) C++ plugin but no embedded resource -> warning saying to upgrage
3) C++ plugin with embedded resource -> files downloaded